plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/stock-indicator/index.ts

```diff
@@ -26,6 +26,7 @@ const blockConfig: BlockConfiguration = {
 	attributes,
 	supports,
 	edit,
+	save: () => null,
```

Comment:
With this line, we will display the `Stock Indicator` block outside the non-blockified `Add to Cart with Options` and fix [this error](https://github.com/woocommerce/woocommerce/issues/53461).